<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,1,20">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,10,1,1,10">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>u` =</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="derivativeTEdit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Analyt expr:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="xyChoiseBox">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
        </property>
        <item>
         <property name="text">
          <string>y =</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>x =</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="analytTEdit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>decimals acc</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="decimalsCountSpin">
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>x0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="x0Spin">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>y0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="y0Spin">
        <property name="decimals">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>xmax</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="xmaxSpin">
        <property name="value">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>step or eps</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="stepSpin">
        <property name="decimals">
         <number>10</number>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="autoStepCheck">
        <property name="text">
         <string>auto</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="calcBtn">
        <property name="text">
         <string>calc</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>results â†“</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,10">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QSpinBox" name="analytAprxSpin">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="analytAprxTableIn">
            <property name="rowCount">
             <number>1</number>
            </property>
            <property name="columnCount">
             <number>2</number>
            </property>
            <row/>
            <column/>
            <column/>
            <item row="0" column="0">
             <property name="text">
              <string>1st approx:</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tableView">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>250</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QChartView" name="graphicsView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>549</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
